framework/src/base/FEProblemBase.C

```diff
@@ -3612,13 +3612,18 @@ FEProblemBase::computeResidualType(const NumericVector<Number>& soln, NumericVec
 void
 FEProblemBase::computeJacobian(NonlinearImplicitSystem & /*sys*/, const NumericVector<Number> & soln, SparseMatrix<Number> & jacobian)
 {
-  computeJacobian(soln, jacobian);
+  computeJacobian(soln, jacobian, Moose::KT_ALL);
 }
 
-
 void
 FEProblemBase::computeJacobian(const NumericVector<Number> & soln, SparseMatrix<Number> & jacobian)
 {
+  computeJacobian(soln, jacobian, Moose::KT_ALL);
```

Comment:
For `FEProblem`, most of people will use `computeJacobian(soln, jacobian, Moose::KT_ALL);` But for `EigenProblem`, we have two routines:  ` computeJacobian(soln, jacobian, Moose::KT_NONEIGEN )`, and  `computeJacobian(soln, jacobian, Moose::KT_EIGEN )`.